daemon/mgr/cri_test.go

```diff
@@ -99,6 +172,33 @@ func TestToCriSandboxState(t *testing.T) {
 
 // Container related unit tests.
 
+func TestParseContainerName(t *testing.T) {
+	format := fmt.Sprintf("%s_${container name}_${sandbox name}_${sandbox namespace}_${sandbox uid}_${attempt times}", kubePrefix)
+
+	// A successful case is omitted here due to the existent test in TestContainerNameRoundTrip.
+	longerNameContainer := "k8s_cname_name_namespace_uid_3_4"
+
+	WrongPrefixContainer := "swarm_cname_name_namespace_uid_3"
+
+	WrongAttemptContainer := "k8s_cname_name_namespace_uid_notInt"
```

Comment:
Actually, in golang, capitalization of the first letter means that the variable is visible outside of the package.

Obviously, we don't need to make it public, so `WrongAttemptContainer` -> `wrongAttemptContainer`, as well as `WrongAttemptErr` .